views/firstLogin/sections/thanks.hbs

```diff
@@ -2,14 +2,14 @@
     <div class="welcomeGreeting">
         <div class="welcomeGreetingImage text-center" id="welcome">
             <div class="content">
-                <img src="/images/firstLogin/welcome_screen.svg">
+                <img alt="{{$t "first_login.img_alt.wavingStudents"}}" src="/images/firstLogin/welcome_screen.svg">
```

Comment:
Isn't the image for presentational purpose only? then I would change the role and hide it from AT.